src/Equinox.DynamoStore/DynamoStore.fs

```diff
@@ -1131,11 +1131,9 @@ type internal Category<'event, 'state, 'context>
         match! store.Reload(log, (streamNam, pos), requireLeader, (codec.TryDecode, isOrigin), ct) with
         | LoadFromTokenResult.Unchanged -> return struct (streamToken, state)
         | LoadFromTokenResult.Found (token', events) -> return token', fold state events }
-    interface Caching.IReloadableCategory<'event, 'state, 'context> with
-        member _.Load(log, _categoryName, _streamId, streamName, _allowStale, requireLeader, ct) =
+    interface ICategory<'event, 'state, 'context> with
+        member _.Load(log, _categoryName, _streamId, streamName, _maxStaleness, requireLeader, ct) =
```

Comment:
This is probably pragmatic, but seeing that every store implementation is ignoring the `_maxStaleness` parameter has me wondering if this is the right interface 🤔

Reply:
The other idea I have in that space is for the layers inward of the cache to have a `LoadOrReload` signature with a `origin: (token*state) voption`, rather than Load+Reload (should that be called `Fetch` ?). ICategory and ICategoryImpl ;)